### ISSUE TITLE
Some download massage

### ISSUE THREAD

**Issue by Christian-CFP** [SeydX/homebridge-fritz-platform#199]
Got this massage in the log since 6.0.5 and FB 7.27

[HOOBS-Server 6A21@Fritz!Box 7590@Fritz!Box 7590@Download] characteristic value expected string and received boolean

**Comment by SeydX** [comment]
> Got this massage in the log since 6.0.5 and FB 7.27
> 
> [HOOBS-Server 6A21@Fritz!Box 7590@Fritz!Box 7590@Download] characteristic value expected string and received boolean

Weird, does the message keep coming? Or did it come only once?

**Comment by Christian-CFP** [comment]
It keeps coming every 5-10min. Sometimes even more often.

**Comment by Christian-CFP** [comment]
e.g.
26.6.2021, 12:26:53 [HOOBS-Server 6A21@Fritz!Box 7590@Fritz!Box 7590@Download] characteristic value expected string and received boolean
26.6.2021, 12:28:23 [HOOBS-Server 6A21@Fritz!Box 7590@Fritz!Box 7590@Download] characteristic value expected string and received boolean
26.6.2021, 12:28:39 [HOOBS-Server 6A21@Fritz!Box 7590@Fritz!Box 7590@Download] characteristic value expected string and received boolean

**Comment by SeydX** [comment]
Can u try v6.0.6 pls

**Comment by Christian-CFP** [comment]
Seems to be good now!

I got some other stuff and will open another issue and write in german.
